pfcwd/main.py

```diff
@@ -62,7 +63,12 @@
 @click.group()
 def cli():
     """ SONiC PFC Watchdog """
+    initialize_db_config()
 
+def initialize_db_config():
+    # Initialize db config here for UT coverage
+    load_db_config()
+    return
```

Comment:
Our code coverage requirement is 80%, so need return here to pass coverage check.

Reply:
Prefer keep code cleaner, and fix the test infra.

Reply:
I am okay to override coverage check in this PR as long as there is an issue tracking the coverage issue.

Reply:
Will improve code and try fix test infra later.

Reply:
I checked the code coverage of some other script, the @click.group() of those script are all not covered, for example piceutil, so will try cover the cli() and if not work will change coverage rate to 50%

<html>
<body>
<!--StartFragment-->



  |   | 69 |   | # This is our main entrypoint - the main 'psuutil' command
  | 1 | 70 |   | @click.group()
  |   | 71 |   | def cli():
  |   | 72 |   | """pcieutil - Command line utility for checking pci device"""
  | 0 | 73 |   | if os.geteuid() != 0:
  | 0 | 74 |   | click.echo("Root privileges are required for this operation")
  | 0 | 75 |   | sys.exit(1)
  |   | 76 |   |  
  |   | 77 |   | # Load platform-specific psuutil class
  | 0 | 78 |   | load_platform_pcieutil()
  |   | 79 |  

<br class="Apple-interchange-newline"><!--EndFragment-->
</body>
</html>

Reply:
This seems a known issue in click: https://github.com/pallets/click/issues/267
Will try use subprocess.check_output to cover code in cli().

Reply:
Revert the code coverage threshold change for force merge.